frontend/src/components/Login.jsx

```diff
@@ -46,12 +61,13 @@ class Login extends Component {
           <h2>For Creators:</h2>
           <FacebookLogin
             appId={process.env.FB_APP_ID}
-            fields="first_name,last_name,email, picture, accounts{instagram_business_account}"
+            fields="first_name,last_name,email, picture, gender, location, link, accounts{instagram_business_account}"
+            //change the scope of the login
+            scope="public_profile,email, user_gender, user_location, user_link, instagram_basic,instagram_manage_comments,manage_pages"
+            autoLoad={true}
```

Comment:
setting autoLoad to true logs in the user right away (when the Login component loads) even without the user clicking the Facebook Login button